libyul/AssemblyStack.cpp

```diff
@@ -113,7 +113,20 @@ bool AssemblyStack::analyzeParsed(Object& _object)
 {
 	solAssert(_object.code, "");
 	_object.analysisInfo = make_shared<AsmAnalysisInfo>();
-	AsmAnalyzer analyzer(*_object.analysisInfo, m_errorReporter, boost::none, languageToDialect(m_language, m_evmVersion));
+
+	set<YulString> objectNames;
+	objectNames.insert(_object.name);
+	for (auto const& subObject: _object.subIndexByName)
+		objectNames.insert(subObject.first);
```

Comment:
Would this have more nested levels?

Reply:
Yes, but that is not yet implemented in the code generator either.